### PR TITLE
Adding stuff for tone page (BLOCKED)

### PR DESCRIPTION
**Pull request by heinburger**

### CODE REVIEW

File: frontend/dev/jsx/layout/container.jsx [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
+import React, {Component} from 'react'
 
-export default class Container extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-
-    }
-  }
+export default class Container extends Component {
```
Comment (tophboogie) [inline frontend/dev/jsx/layout/container.jsx]: what is the benefit of doing this vs "extends React.Component"

Comment (heinburger) [inline frontend/dev/jsx/layout/container.jsx]: @tophboogie oops missed this - so I think if we do it with the destructuring, we can eliminate the unused exports (createClass, Children, etc.) from react with webpack (not sure if this is only webpack 2 or not, but it is supposed to be smart enough to do that) we still need the `import React..` for the jsx transpiling, but if you don't use the `React` reference anywhere, webpack only uses it for the jsx in dev and it is not part of the bundle - this way we only get what we are using

Comment (tophboogie) [inline frontend/dev/jsx/layout/container.jsx]: OOoooooooohhhhhh IC

**Comment by heinburger** [comment]
closing due to refocus

**Pull request closed without merging**